client/src/pages/ProjectLeaderDashboard.js

```diff
@@ -62,7 +62,7 @@ const ProjectLeaderDashboard = () => {
           if (!item.userId.name) {
             item.userId.name = {};
             item.userId.name.firstName = item.userId.firstName;
-            item.userId.name.lastName = item.userId.lastName; 
+            item.userId.name.lastName = item.userId.lastName;
```

Comment:
I would review the function that contains this change and see if we can not mutate outside data, but rather create copies.

Reply:
@nickbeaird The data from DB contains some legacy records that should be processed before further use. Original outside data is useless because it breaks rendering.
@Jonathanko52 skip this comment.